### DISCUSSION TITLE
Question: Why virtio scsi & vhost scsi is not supported in cloud hypervisor?

### DISCUSSION THREAD

**Discussion by optimistyzy** [cloud-hypervisor/cloud-hypervisor#3368] [Q&A]
Would like to know why virtio scsi and vhost scsi is not supported in Cloud Hypervisor? Is virtio-blk/vhost block enough to cover most of the cases while using virtio scsi/vhost scsi?

**Comment by iggy** [comment]
This should probably be a [discussion](https://github.com/cloud-hypervisor/cloud-hypervisor/discussions)

**Comment by sboeuf** [comment]
Yes most use cases should be covered by `virtio-block` or `vhost-user-block`.
